### PR TITLE
[py] TChannel should not be a singleton

### PR DESCRIPTION
**Pull request by abhinav**
@blampe @breerly @junchaowu

### CODE REVIEW

**Comment by junchaowu** [comment]
:ship:

**Comment by breerly** [comment]
:ship: :it:

File: python/tchannel/tornado/tchannel.py [hunk 1]
```diff
@@ -54,23 +53,6 @@ class TChannel(object):
     separate instances, use the ``ignore_singleton`` argument.
     """
 
-    # We don't want to duplicate outgoing connections, so all instances of this
-    # class will be a singleton.
-    _singleton = threading.local()
-
-    class __metaclass__(type):
```
Comment (breerly) [inline python/tchannel/tornado/tchannel.py]: holy fancy jesus :P

Comment (abhinav) [inline python/tchannel/tornado/tchannel.py]: sorry :disappointed:

**Comment by blampe** [comment]
rip bro

**Pull request merged**